src/main/java/com/cachinginapp/enterprise/CachingInController.java

```diff
@@ -101,7 +116,7 @@ public Cache createCache(@RequestBody Cache cache) {
         try {
             newCache = cacheService.save(cache);
         } catch (Exception e) {
-            //TODO add logging
+            logger.log(Level.WARNING, "Your cache was not created.");
```

Comment:
added logger and try catch in case of exception

Reply:
Ah, great.